force-app/main/default/classes/StringGenerator.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class StringGenerator {
+
+    private List<String> my_list = new List<String>();
+
+    public StringGenerator( List<String> my_list_by_user )
+    {
+        my_list = my_list_by_user;
+    }
+    
+    public List<String> returnRandomString()
+    {
+        List<Integer> randomIndexList = new List<integer>();
+        
+        List<String> randomStringList = new List<String>();
+        
+        Integer randomIndex;
+        
+        while( randomIndexList.Size() < my_list.Size() )
+        {
+            randomIndex = integer.valueof((Math.random() * my_list.Size() ));
+            
+            if( !( randomIndexList.contains(randomIndex) ) )
+            {
+                randomIndexList.add( randomIndex );
+                randomStringList.add( my_list[randomIndex] );
+            }
+        }
+        return randomStringList;
+    }
+}
```

Comment:
the problem statement required to have 2 methods i-e

1) a random string (completely randomly chosen from the list passed as input to the class) 

2) a uniquely random string (one string should not repeat until the cycle is complete).

Reply:
I see, I have missed one method randomString() which is different from uniquelyRandomString()
I'll add that function, and will name the methods accordingly